packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-p2p-card/cashier-onboarding-p2p-card.tsx

```diff
@@ -43,8 +35,6 @@ const CashierOnboardingP2PCard: React.FC = observer(() => {
         history.push('/cashier/p2p');
     };
 
-    if (!should_show_p2p_card) return null;
```

Comment:
removed any logic that checks whether or not we should display p2p in the toggle-menu-drawer under cashier or under cashier tabs. This is because we already handle if the user can view p2p or not within the standalone application already. Also we should still show p2p exists, even if the user can't use it, this was an earlier suggestion from jy a long time back, since how will users know certain applications exist within deriv if we never see it